docs/design/flow-traffic-mirroring/flow-traffic-mirroring.md

```diff
@@ -0,0 +1,172 @@
+# Flow traffic mirroring
+
+## Idea
+There must be an ability to mirror traffic on a given flow and send it to a specific switch+port+vlan.
+
+## Model
+FlowPath has a list of FlowMirrorPoints, which will contain MirrorGroupId for multicasting on the switch. 
+FlowMirrorPoint itself has a list of MirrorFlowPaths. MirrorFlowPaths contains the endpoint 
+where the traffic should be mirrored and other necessary information to build the path.
+
+![DB model](./model.png "DB model")
+
+If the receiver point is on the mirroring source switch, then no segments will be built for such a MirrorFlowPath, 
+and we will work with it like the paths in one switch flow.
+
+## API
+* Create mirror point:
+  
+  `PUT /flows/{flow_id}/mirror`
+  
+  payload:
+  ```
+  {
+      "mirror_point_id": string,
```

Comment:
What is the "mirror_point_id" where client will get it?

Reply:
The user must provide this identifier.

Reply:
Mention somewhere constraints for this field - max/min length, level of uniqueness (unique across whole system | unique across flow / path | not unique at all etc). Or move responsibility for creation of this identifier from client to kilda system. In this case all such constrains will be obviously defined in kilda code.

Reply:
The description has been added to the document. Also, request filtering will be added to the code.